File: 프로그래머스/lv2/42584. 주식가격/주식가격.js
function solution(prices) {
    //ver 1 : 이중 for 문
    const result = [];
    for(let i = 0; i < prices.length; i++){
        for(let j = i + 1; j < prices.length; j++){
            if(prices[i] > prices[j]){
                result.push(j - i);
                break;
            }else if(prices[i] <= prices[j] && j === prices.length -1){
                result.push(j - i);
            }
        }
    }
    result.push(0)
    return result
}

File: 프로그래머스/lv2/42584. 주식가격/주식가격.py
def solution(prices):
    result = [0] * len(prices);
    for i in range(len(prices)):
        for j in range(i + 1, len(prices)):
            if prices[i] <= prices[j]:
                result[i] += 1;
            else:
                result[i] += 1
                break;
    return result

